notification-portlet-webapp/src/main/java/org/jasig/portlet/notice/NotifacationApplication.java

```diff
@@ -23,11 +23,11 @@
 import org.springframework.context.annotation.ImportResource;
 
 @SpringBootApplication
-@ImportResource("classpath:/context/*.xml") // Legacy, XML-based beans
+@ImportResource({"classpath:/context/*.xml","classpath:/properties/contextOverrides/*.xml"}) // Legacy, XML-based beans
```

Comment:
The code looks good, could we rename the file from `NotifacationApplication.java` to `NotificationApplication.java`?

Reply:
Sounds good